### PR TITLE
Added our-community and our-impact section

### LINKED ISSUE #78: Adding "Our Impact" and "Our Community" Section
**Add Our Impact and Our Community Section**
Hi @codewithvk I am Tanish . As this website is in progress , so I want to contribute in this program by adding above two sections under this issue . Also I have gone through most of opened issues and no one asked you to work on above 2 sections .

**I Want to add interactive sections to this web page**
As you have some a sample preview in video about how these sections look like , I want to make something like that amazing designs sections .

**Additional context**
I have done web development from coding ninjas and have certificate for the same uploaded on linkedIn and also made some amazing projects like Music player project etc (uploaded on linkedIn) . Also I am learning advance web development (back end) currently , so I can also contribute further in this project after this issue as well . I am very hardworking and scored 94.2% in 12th . Before this project I have merged one pull request in Awesome JavaScript projects , solved one issue in Public Lab repository .
LinkedIn profile link - https://www.linkedin.com/in/tanish-a54b771b3
Looking forward for working for this project .
Thank You
Regards

### PR DESCRIPTION
**Pull request by SharmaTanish**
# Description

I have added our-community and our-impact section under above issue using react .
@codewithvk please have a look !

Closes #78

## Type of change

Please delete options that are not relevant.
- [x] New feature (non-breaking change which adds functionality)

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] Any dependent changes have been merged and published in downstream modules
## Screenshots
Screenshot (1602)

Screenshot (1603)

### CODE REVIEW

File: frontend/src/App.js [hunk 1]
```diff
@@ -3,11 +3,21 @@ import { Fragment } from "react";
...
 import './App.css';
+import React, { PureComponent } from 'react';
+import { PieChart, Pie, Sector, Cell, ResponsiveContainer } from 'recharts';
 // Pages Import
-
 import HomePage from "./Pages/Homepage"
 import Footer from "./Components/Footer/footer"
 const App = () => {
+ 
+  const data01 = [
```
Comment (codewithvk) [inline frontend/src/App.js:13]: For fake/Test_data , You can create new file on `frontend/src/Test_data/` and fetch from there!

File: frontend/src/App.js [hunk 2]
```diff
@@ -19,7 +29,70 @@ const App = () => {
           <Footer/>
         </Router>
       </div>
+
+
+      <h1 className="our_impact" ><b>OUR IMPACT</b></h1>
```
Comment (codewithvk) [inline frontend/src/App.js:34]: Hey @SharmaTanish , Please make a separate component for that, for reference you can see `frontend/src/Components/Navbar/`.

App.js Use only for a component call with the router.

**Review by codewithvk — CHANGES_REQUESTED** [review]
Hey @SharmaTanish , Please make the above changes!
If you have any query you can react me out by discord or here.

**Comment by SharmaTanish** [comment]
Hi @codewithvk I have successfully made the changes you have requested ! Please have a look ?

**Comment by SharmaTanish** [comment]
Thank you @codewithvk for your guidance and support 😀!

**Pull request merged**